examples/user_guide/Network_Graphs.ipynb

```diff
@@ -0,0 +1,335 @@
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%%opts Graph [tools=['hover']]\n",
+    "import networkx as nx\n",
+    "G = nx.karate_club_graph()\n",
+    "hv.Graph.from_networkx(G, nx.layout.circular_layout).redim.range(**padding)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### Animating graphs"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%%opts Graph {+framewise}\n",
+    "layouts = [nx.layout.circular_layout, nx.layout.fruchterman_reingold_layout, nx.layout.spectral_layout,\n",
+    "           nx.layout.spring_layout, nx.layout.random_layout]\n",
+    "\n",
+    "G = nx.karate_club_graph()\n",
+    "hv.HoloMap({l.__name__: hv.Graph.from_networkx(G, l) for l in layouts},\n",
+    "           kdims=['Layout'])"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Real world graphs"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "As a final example let's look at a slightly larger graph. We will load a dataset of a Facebook network consisting a number of friendship groups identified by their ``'circle'``. We will load the edge and node data using pandas and then color each node by their friendship group using many of the things we learned above."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%opts Nodes Graph [width=800 height=800 xaxis=None yaxis=None tools=['hover']]"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%%opts Graph [color_index='circle']\n",
+    "%%opts Graph (node_size=10 edge_line_width=1)\n",
+    "colors = ['#000000']+hv.Cycle('Category20').values\n",
+    "edges_df = pd.read_csv('../assets/fb_edges.csv')\n",
+    "fb_nodes = hv.Nodes(pd.read_csv('../assets/fb_nodes.csv')).sort()\n",
+    "fb_graph = hv.Graph((edges_df, fb_nodes), label='Facebook Circles')\n",
+    "fb_graph = fb_graph.redim.range(x=(-0.05, 1.05), y=(-0.05, 1.05)).opts(style=dict(cmap=colors))\n",
+    "fb_graph"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Bundling graphs"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The datashader library provides algorithms for bundling the edges of a graph and HoloViews provides convenient wrappers around the libraries."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from holoviews.operation.datashader import datashade, bundle_graph\n",
+    "bundled = bundle_graph(fb_graph)\n",
+    "bundled"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Datashading graphs"
+   ]
```

Comment:
This plot isn't yet compelling.  For one thing, the colorized edges don't really show up, which is partly because colors of fine lines are just never very visible, and partly because they are using the fire colormap, which is an inappropriate choice against a white background.  Here the strongest bundles are colored yellow, which is very hard to see against white, thereby deemphasizing precisely the bits that should be most emphasized.  I really think we need to get rid of the default "fire" colormap for datashader in HoloViews; it's almost never the right choice for the default white backgrounds.  

In any case, this example isn't yet very compelling, perhaps because the data volume isn't high enough to need datashader in the first place.  Maybe show the unbundled graph too, where the color shading can help make sense of the tangled thicket around the main clump of nodes?

Reply:
Since this is a user guide I don't think a larger dataset is workable. I agree about changing the color scheme, but do you think a larger dataset is required? I'd like to avoid switching to synthetic data.

Reply:
Well, try the non-bundled case; the effect of datashader's coloring should be clear there.

Reply:
It just looks very messy unfortunately, I've switched to 'linear' normalization, which means that the paths are mostly black. If we can get the nodes to line up I think this is reasonable.